### PR TITLE
Updates to install documentation and one typo fixed

### PR DESCRIPTION
**Pull request by waynebeech**
added/corrected comments/instructions to INSTALL-centos.md file

Fixed a typo (nost instead of host) in example-parameters.json file

### CODE REVIEW

File: INSTALL-centos.md [hunk 1]
```diff
@@ -309,3 +321,7 @@ PORTAL_URN=urn:publicid:IDN+${AUTHORITY}+authority+portal
 geni-add-trusted-tool -d portal -u portal -p portal --host localhost \
     'GENI Portal' "${PORTAL_URN}"
 ```
+
+Open up Firewall if necessary
+-----------------------------
+If your machine is running firewall software it may be necessary for you to add rules to allow connections to the clearinghouse.
```
Comment (tcmitchell) [inline INSTALL-centos.md]: Do you know what ports have to be open?

Comment (waynebeech) [inline INSTALL-centos.md]: i was going to double check.  i assume 8444 is the only one other than if you want 22 80 443 open?

wayne

File: INSTALL-centos.md [hunk 2]
```diff
@@ -74,6 +80,12 @@ sudo cp /etc/geni-chapi/example-parameters.json /etc/geni-chapi/parameters.json
 Edit `/etc/geni-chapi/parameters.json`:
 * Make sure to set `db_host` and `ch_host`!!
 
+Do this by adding a line of the format:
+
+"default" : "hostname.domain.tld"
```
Comment (tcmitchell) [inline INSTALL-centos.md]: Enclose this in "```" (three backticks) before and after like so to show it as a code example like this:

    ```
    "default" : "hostname.domain.tld"
    ```

File: INSTALL-centos.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 Installation on CentOS 7
 ========================
 
+Introduction
+------------
+These instructions are for installing the GENI Clearing house.  Information on the Clearinghouse can be found at:
```
Comment (tcmitchell) [inline INSTALL-centos.md]: Clearinghouse should be one word in the first occurrence.

**Comment by tcmitchell** [comment]
Thanks for submitting these changes, they look great. I added some very minor comments for you to consider. Let us know when you're done and we'll merge these changes in.

File: INSTALL-centos.md (hunk 3 above)
Comment (waynebeech) [inline INSTALL-centos.md]: ok, want me to update and then ask for a pull again?  do you need to reject the current request or anything like that?

wayne

Comment (tcmitchell) [inline INSTALL-centos.md]: If you make the changes and commit them to your develop branch they will automatically update in this pull request.

**Pull request merged**